interfaces.go

```diff
@@ -118,12 +118,14 @@ type ChainSyncReader interface {
 
 // CallMsg contains parameters for contract calls.
 type CallMsg struct {
-	From     common.Address  // the sender of the 'transaction'
-	To       *common.Address // the destination contract (nil for contract creation)
-	Gas      uint64          // if 0, the call executes with near-infinite gas
-	GasPrice *big.Int        // kei <-> gas exchange ratio
-	Value    *big.Int        // amount of kei sent along with the call
-	Data     []byte          // input data, usually an ABI-encoded contract method invocation
+	From      common.Address  // the sender of the 'transaction'
+	To        *common.Address // the destination contract (nil for contract creation)
+	Gas       uint64          // if 0, the call executes with near-infinite gas
+	GasPrice  *big.Int        // kei <-> gas exchange ratio
+	GasFeeCap *big.Int        // EIP-1559 fee cap per gas.
+	GasTipCap *big.Int        // EIP-1559 tip per gas.
```

Comment:
What about adding a comment that it is same word with `MaxGasFeeCap` and `MasGasFeeCap`?

Reply:
Fixed in https://github.com/kaiachain/kaia/pull/29/commits/cbdca4769b5253b583410032e6285a80ea101a7e.